ios/Framework/Themes/Default/Rating/RatingView_default.like.swift

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+import UIKit
+
+public class RatingLikeView_default: BaseScreenletView, RatingViewModel {
```

Comment:
The name of the class should be the same of the swift file.

For sub-themes, maybe it's better to use "_default-subname" instead of "_default.subname"